.github/workflows/oppiabot.yml

```
@@ -0,0 +1,21 @@
+# Contains jobs corresponding to Oppiabot.
+
+name: Oppiabot
+
+on:
+  pull_request_target:
+    branches:
+      - develop
+
+jobs:
+  oppiabot:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: CLA Check
+        uses: oppia/oppiabot@1.2.0
+        env:
+          SHEETS_TOKEN: ${{ secrets.SHEETS_TOKEN }}
+          SHEETS_CRED: ${{ secrets.SHEETS_CRED }}
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          SPREADSHEET_ID: ${{ secrets.SPREADSHEET_ID }}
```

```suggestion
        uses: oppia/oppiabot@1.3.0
        with:
          repo-token: ${{secrets.GITHUB_TOKEN}}
        env:
          SHEETS_TOKEN: ${{ secrets.SHEETS_TOKEN }}
          SHEETS_CRED: ${{ secrets.SHEETS_CRED }}
          SPREADSHEET_ID: ${{ secrets.SPREADSHEET_ID }}
```

Oppiabot made a new release with some changes to CLA Check Github Action

---

Done